cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -369,7 +369,10 @@ impl pallet_assets::Config<ForeignAssetsInstance> for Runtime {
 	type AssetIdParameter = MultiLocationForAssetId;
 	type Currency = Balances;
 	type CreateOrigin = ForeignCreators<
-		(FromSiblingParachain<parachain_info::Pallet<Runtime>>,),
+		(
+			FromSiblingParachain<parachain_info::Pallet<Runtime>>,
+			snowbridge_router_primitives::inbound::FromEthereumGlobalConsensus<EthereumLocation>,
```

Comment:
as I said above, I think this can be replaced with `FromNetwork<UniversalLocation, EthereumNetwork>` implementation